server/src/test/resources/csars/yaml/valid/lamp-csar/scripts/start_myphpapp.php

```diff
@@ -0,0 +1,82 @@
+<?php
+// include the credentials to connect to the db
+//include_once "mysql-credentials.php";
+
+$db_host = "127.0.0.1";
```

Comment:
Why did you put everything into one file? Two files were better?
How do you plan to integrate the inputs? Sorry if it still work in progress.